Dashboard - unstyled.

felipesere opened the pull request:
There are still plenty of todos on the dashboard:

* [x] group leads should be able to delete a questionnaire
* [x] group leads should be able to do download a CSV of a questionnaire
* [x] group leads should be able to go to the summary of a questionnaire
* [x] users should not see advice requests they have already submitted
* [x] users should be able to distinguish between people who have submitted advice for them those who haven't

RabeaGleissner commented on lib/advisor/core/questionnaire/creator.ex:
```diff
@@ -25,6 +26,6 @@ defmodule Advisor.Core.Questionnaire.Creator do
 
   defp to_advisory(%{advisor_id: advisor_id, id: id}) do
-    %Advisory{advisor: People.find_by_id(advisor_id), advice_id: id}
+    %Advisory{advisor: People.find_by_id(advisor_id), advice_id: id, id: id}
```
Just like the TOOD above says :)

RabeaGleissner commented on test/core/advice_test.exs:
```diff
@@ -0,0 +1,34 @@
+    assert Advice.completed?(christoph, length(@questions))
+  end
+
+  test "cann tell if advice has not been answered fully" do
```
too many "n"s in the word "can"

RabeaGleissner commented on test/web/controllers/dashboard_page_test.exs:
```diff
@@ -0,0 +1,139 @@
+    |> still_has_to_give_me_advice("Sarah Johnston")
+  end
+
+  @tag :skip
```
did you mean to skip this? The functionality is there now, isn't it?

felipesere commented:
@RabeaGleissner @fabientownsend I've updated the PR
I managed to remove the advisory record and extracted helper methods into two modules `ThroughTheCore` and `ThroughTheWeb`.
Any remaining feedback?

RabeaGleissner commented:
Looks good to me! Can't think of any other improvements...
